### DISCUSSION TITLE
ExpirationTime

### DISCUSSION THREAD

**Discussion by semce2** [Windos/BurntToast#186] [Q&A]
I am unable to get the expirationtime parameter to work. Seems to always default to the system timeout.  I am using (Get-Date).AddSeconds(7).  I have been unable to find help for this parameter.  Thanks

**Comment by Windos** [comment]
Hey there @semce2

Sorry for the delay in responding, I'm sure you've moved on since this posting this but just to ensure there's an answer here.

The "help" for that parameter can be viewed with this command:

```powershell
Get-Help New-BurntToastNotification -Parameter ExpirationTime
```

```
-ExpirationTime <DateTime>
    The time after which the notification is no longer relevant and should be removed from the Action Center.

    Required?                    false
    Position?                    named
    Default value
    Accept pipeline input?       false
    Accept wildcard characters?  false
```

The key is that expiration time doesn't affect how long it shows on screen (which is what you'd _think_ it affects), instead it's a hint to Windows to tell it when it should be removed from the Action Center (where missed notifications live.)

An example use case for this is sending out a notification saying that there's muffins in the lunchroom. Let's say you send this out at noon you might set the expiration time to 1PM, so that if someone missed it they don't see it after lunch, or the next day, and get confused when there's no muffins. (Yes, the example is contrived, but it seems to land with most people 😅)

Better docs is part of what I'm working on for the BurntToast v1 release and so the usage of this parameter _should_ be clearer in the future.

**Comment by Windos** [comment]
Hello, and thank you for your contribution to BurntToast.

Today marks the official release of [BurntToast v1.0.0](https://github.com/Windos/BurntToast/releases/tag/v1.0.0). As part of this major milestone and to ensure the project starts its next chapter with a clean slate, I am declaring "issue bankruptcy."

This decision is explained in more detail in the official announcement post: [Ten Years Toasting: BurntToast Hits the Big v1!](https://toastit.dev/2025/07/22/burnttoast-v1/)

As a result, I am closing all outstanding issues and pull requests (and discussions), including this one. Please do not interpret this as a lack of interest in the problem, suggestion, or topic you've raised. This is a necessary step to clear a very old backlog and focus development efforts on the current state of the module.

**If this topic is still relevant to you:**

1.  Please **open a new discussion**. You are welcome to link back to this one for historical context.

I hope this reset will allow me to be more responsive to community feedback going forward.

Thank you for your understanding and for being part of the BurntToast journey.
